README.md

```diff
@@ -1,70 +1,50 @@
-# Getting Started with Create React App
+# _Anime Finder App_
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+An App that simulates a classic GameBoy style Pokemon battle.
 
-## Available Scripts
+---
 
-In the project directory, you can run:
+## Authors
 
-### `npm start`
+Kortney Foley
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+---
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Technologies Used
 
-### `npm test`
+-   HTML
+-   JavaScript
+-   CSS
+-   React
```

Comment:
You may want to add classnames here since that's an npm package that's installed.